cmd/knit/README.md

```diff
@@ -0,0 +1,140 @@
+IRQ 125 [                xhci_hcd]: can run on [0 1 2 3]
+IRQ 126 [                 nvme0q0]: can run on [0 1 2 3]
+IRQ 127 [               enp0s31f6]: can run on [0 1 2 3]
+IRQ 128 [                 nvme0q1]: can run on [0]
+IRQ 129 [                 nvme0q2]: can run on [1]
+IRQ 130 [                 nvme0q3]: can run on [2]
+IRQ 131 [                 nvme0q4]: can run on [3]
+IRQ 132 [                    i915]: can run on [0 1 2 3]
+IRQ 133 [                  mei_me]: can run on [0 1 2 3]
+IRQ 134 [                 iwlwifi]: can run on [0 1 2 3]
+IRQ 135 [     snd_hda_intel:card0]: can run on [0 1 2 3]
+IRQ 136 [              rmi4_smbus]: can run on [0 1 2 3]
+IRQ 137 [            rmi4-00.fn34]: can run on [0 1 2 3]
+IRQ 138 [            rmi4-00.fn01]: can run on [0 1 2 3]
+IRQ 139 [            rmi4-00.fn03]: can run on [0 1 2 3]
+IRQ 140 [            rmi4-00.fn11]: can run on [0 1 2 3]
+IRQ 141 [            rmi4-00.fn11]: can run on [0 1 2 3]
+IRQ 142 [            rmi4-00.fn30]: can run on [0 1 2 3]
+```
+
+Checking which IRQ will be served on CPUs #1 and #2
+```bash
+$ knit irqaff -C 1,2
+IRQ   0 [                        ]: can run on [1 2]
+IRQ   1 [                   i8042]: can run on [1 2]
+IRQ   2 [                        ]: can run on [1 2]
+IRQ   3 [                        ]: can run on [1 2]
+IRQ   4 [                        ]: can run on [1 2]
+IRQ   5 [                        ]: can run on [1 2]
+IRQ   6 [                        ]: can run on [1 2]
+IRQ   7 [                        ]: can run on [1 2]
+IRQ   8 [                    rtc0]: can run on [1 2]
+IRQ   9 [                    acpi]: can run on [1 2]
+IRQ  10 [                        ]: can run on [1 2]
+IRQ  11 [                        ]: can run on [1 2]
+IRQ  12 [                   i8042]: can run on [1 2]
+IRQ  13 [                        ]: can run on [1 2]
+IRQ  14 [                        ]: can run on [1 2]
+IRQ  15 [                        ]: can run on [1 2]
+IRQ  16 [              i801_smbus]: can run on [1 2]
+IRQ 125 [                xhci_hcd]: can run on [1 2]
+IRQ 126 [                 nvme0q0]: can run on [1 2]
+IRQ 127 [               enp0s31f6]: can run on [1 2]
+IRQ 129 [                 nvme0q2]: can run on [1]
+IRQ 130 [                 nvme0q3]: can run on [2]
+IRQ 132 [                    i915]: can run on [1 2]
+IRQ 133 [                  mei_me]: can run on [1 2]
+IRQ 134 [                 iwlwifi]: can run on [1 2]
+IRQ 135 [     snd_hda_intel:card0]: can run on [1 2]
+IRQ 136 [              rmi4_smbus]: can run on [1 2]
+IRQ 137 [            rmi4-00.fn34]: can run on [1 2]
+IRQ 138 [            rmi4-00.fn01]: can run on [1 2]
+IRQ 139 [            rmi4-00.fn03]: can run on [1 2]
+IRQ 140 [            rmi4-00.fn11]: can run on [1 2]
+IRQ 141 [            rmi4-00.fn11]: can run on [1 2]
+IRQ 142 [            rmi4-00.fn30]: can run on [1 2]
+```
+
+Checking softirqs affinity. All CPUs served softirqs.
```

Comment:
Is there a reason the format is different?

Reply:
yes: I couldn't find yet where softirqs affinities are exposed - if they are exposed at all. So we can just check after the fact which cpus served softirqs peeking at the counters.